package.json

```diff
@@ -23,7 +23,7 @@
     "json-schema": "^0.2.3",
```

Comment:
btw, I think we can safely move the `bluebird` (a promise library) dependency to `devDependencies` since it is only used in tests.